operator/cmd/syncclusterconfig/sync.go

```diff
@@ -377,9 +384,14 @@ func (s *Syncer) Sync(ctx context.Context, desired map[string]any, usersTXT map[
 	if err != nil {
 		return nil, err
 	}
+	latest, err := s.Client.Config(ctx, true)
```

Comment:
this does not contain secrets 🤔 if it's calling admin API. So it won't detect changes in secret fields

Reply:
This will definitely be an issue...
we could add another annotation based on the hash from the secrets?